_studio/mfx_lib/decode/h265/src/mfx_h265_dec_decode.cpp

```diff
@@ -230,6 +231,58 @@ mfxStatus VideoDECODEH265::Init(mfxVideoParam *par)
 
     int32_t useInternal = (MFX_PLATFORM_SOFTWARE == m_platform) ?
         (m_vPar.IOPattern & MFX_IOPATTERN_OUT_VIDEO_MEMORY) : (m_vPar.IOPattern & MFX_IOPATTERN_OUT_SYSTEM_MEMORY);
+#ifndef MFX_DEC_VIDEO_POSTPROCESS_DISABLE
+    mfxExtDecVideoProcessing * videoProcessing = (mfxExtDecVideoProcessing *)GetExtendedBuffer(par->ExtParam, par->NumExtParam, MFX_EXTBUFF_DEC_VIDEO_PROCESSING);
+    /* There are following conditions for post processing via HW fixed function engine:
+     * (1): Progressive only for ICL and Before, interlace is supported for ICL following patforms
+     * (2): Supported on SKL (Core) and APL (Atom) platform and above
+     * (3): Only video memory supported (so, OPAQ memory does not supported!)
+     * */
+    if (videoProcessing)
+    {
+        MFX_CHECK(m_core->GetHWType() >= MFX_HW_SCL &&
```

Comment:
Does feature works starting from "MFX_HW_SCL" ?

Reply:
Yes, the feature can work from SCL.